web/client/actions/security.js

```diff
@@ -168,3 +173,50 @@ export function verifySession() {
 }
 
 export const checkLoggedUser = () => ({type: CHECK_LOGGED_USER});
+
+/**
+ * set status to show modal for inserting credentials
+ * @param {boolean} status
+ */
+export const setShowModalStatus = (status) => {
+    return {
+        status,
+        type: SET_SHOW_MODAL_STATUS
+    };
+};
+/**
+ * set protected services to request to provide username and password
+ * @param {object[]} protectedServices
+ */
+export const setProtectedServices = (protectedServices) => {
+    return {
+        protectedServices,
+        type: SET_PROTECTED_SERVICES
+    };
+};
```

Comment:
These two actions seems specific for the `SecurityPopup` plugin but they are inside the security actions.

Reply:
@MV88 please ignore this comment

Reply:
i did not want to create a securityPopop folder for just these actions but since the plugin is about security it might be good to have theme here